src/org/authInfo.ts

```diff
@@ -26,8 +26,8 @@ import {
   Nullable,
   Optional,
 } from '@salesforce/ts-types';
-import { OAuth2Config, OAuth2, TokenResponse } from 'jsforce';
-import Transport from 'jsforce/lib/transport';
+import { OAuth2Config, OAuth2, TokenResponse } from '@jsforce/jsforce-node';
+import Transport from '@jsforce/jsforce-node/lib/transport';
```

Comment:
Same comment as for `deviceOauthService`.  Also, seems like we can combine multiple lines of imports into 1 statement.